server/plugin/command.go

```diff
@@ -459,7 +458,7 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 		found, foundErr := p.checkIfConfiguredWebhookExists(ctx, githubClient, repo, owner)
 		if foundErr != nil {
 			if strings.Contains(foundErr.Error(), "404 Not Found") {
-				return errorWebhookToUser
+				return subOrgMsg
 			}
 			return errors.Wrap(foundErr, "failed to get the list of webhooks").Error()
 		}
```

Comment:
In the case where the user has access to check webhooks but there isn't one, do we get an error and 404 here? I wonder if there's a way to know if the user has permissions or not

Reply:
@mickmister In the above case, we get no error and the value of `found` as false.